saleor/product/migrations/0125_productvariant_default.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.1 on 2020-09-17 15:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Should we migrete old data?

Reply:
Was thinking about it but it wasn't in requirements and we don't have any logic for a product without default variant. For now (in this PR) the state is - when you create a new variant and there is no default, it sets itself as default.

Reply:
@patrys please advise if we need the data migration.

Reply:
We need to set the first variant as default, the API has to return something and it's better to do this once instead of implementing costly runtime fallbacks

Reply:
Is `default_variant` required?